__tests__/app.test.js

```diff
@@ -83,6 +83,42 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  test("200: responds with an array of all articles", () => {
```

Comment:
Really great test! You've got the length check and your assertion is nice and flexible! And for expecting your data on a meaningful key!